modules/exploits/linux/ssh/solarwinds_lem_exec.rb

```diff
@@ -103,60 +108,61 @@ def exploit
       return
     end
 
-    if ssh
```

Comment:
The entire remainder of this method was wrapped in this `if ssh` conditional with no `else`. I took the opportunity to return early with `return unless ssh` instead.

Reply:
It looks like it returns early in most scenarios that `ssh` would be set to nil as well 🤔